plugins/main/public/components/common/welcome/components/top_packages_table/top_packages_table.tsx

```diff
@@ -0,0 +1,111 @@
+/*
+ * Wazuh app - React component building the welcome screen of an agent.
+ * version, OS, registration date, last keep alive.
+ *
+ * Copyright (C) 2015-2022 Wazuh, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * Find more information about this on the LICENSE file.
+ */
+
+import React, { useState, useEffect, Fragment } from 'react';
+import {
+  EuiBasicTable,
+  EuiFlexItem,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  EuiButtonIcon,
+  EuiFlexGroup,
+  EuiToolTip,
+  EuiLink,
+} from '@elastic/eui';
+// @ts-ignore
+import { getDataPlugin } from '../../../../../kibana-services';
+import { vulnerabilityDetection } from '../../../../../utils/applications';
+import {
+  PatternDataSourceFilterManager,
+  FILTER_OPERATOR,
+} from '../../../data-source';
+import { WzLink } from '../../../../../components/wz-link/wz-link';
+
+export function useTimeFilter() {
+  const { timefilter } = getDataPlugin().query.timefilter;
+  const [timeFilter, setTimeFilter] = useState(timefilter.getTime());
+  useEffect(() => {
+    const subscription = timefilter
+      .getTimeUpdate$()
+      .subscribe(() => setTimeFilter(timefilter.getTime()));
+    return () => {
+      subscription.unsubscribe();
+    };
+  }, []);
+  return timeFilter;
+}
+
+export function VulsTopPackageTable({ agentId, items, indexPatternId }) {
+  const [sort, setSort] = useState({
+    field: 'doc_count',
+    direction: 'desc',
+  });
+  const [data, setData] = useState([]);
+  const timeFilter = useTimeFilter();
```

Comment:
This code is not being used, so it can be removed.

Reply:
Solved in https://github.com/wazuh/wazuh-dashboard-plugins/pull/7085/commits/fc6f1779f0b53f7fb1018d1499af7137c8819f36